.travis.yml

```diff
@@ -1,17 +1,23 @@
 cache: bundler
 sudo: false
 rvm:
-  - 2.2
-  - 2.3
```

Comment:
We should probably add `required_ruby_version ">= 2.4"` to the gemspec.

Reply:
Good call. This has been added in 317e33abadf6383be8f2a5a24f114eed2fc7057a .